hashicorp/dev-portal
# feat: tweak collection progress group appearance

## 🔗 Relevant links

- [Preview link][preview] 🔎
- [Asana task][task] 🎟️

## 🗒️ What

Tweaks Collection Progress Group appearance to match recent revisions.

This change could be summarized as:
- Unauthenticated users now have a tweaked appearance for collection progress elements, with a light gray background
- ~Authenticated users will now always see the `x/x tutorials` + progress bar for collection progress elements~
    - We've since reverted this change, behaviour is the same as before

## 🤷 Why

To address VQA-oriented feedback and improvements on User Progress features.

## 🛠️ How

- chore: update height in sidebar to 36px
- feat: update collection-progress-status-seciton bg
- chore: update use of collection-progress-status-section
- ~feat: update collection-card progress to match~

## 📸 Design Screenshots

🎨 [Figma spec][figma]

### Collection progress in sidebar

---

#### Unauthenticated

##### Before

<img width="1684" alt="sidebar-unauth-before" src="https://user-images.githubusercontent.com/4624598/195153553-de68d48a-2515-47e3-bfe2-af0b2e49cbe7.png">

##### After

<img width="1684" alt="sidebar-unauth-after" src="https://user-images.githubusercontent.com/4624598/195153635-f769c076-ec0b-4a0c-8e2d-d1ec83e0ef28.png">

### Collection progress in Collection meta

---

#### Unauthenticated

##### Before

<img width="1681" alt="meta-unauth-before" src="https://user-images.githubusercontent.com/4624598/195153996-724dbab8-51c3-4bf8-92e8-2a212053b918.png">

##### After

<img width="1684" alt="meta-unauth-after" src="https://user-images.githubusercontent.com/4624598/195154006-e408fc86-155d-476a-90a7-d879a54a4143.png">

## 🧪 Testing

- [ ] Confirm that Collection Progress in sidebar is updated as expected
    - Visit a tutorial page, such as [/consul/tutorials/get-started-hcp/hcp-gs-deploy][/consul/tutorials/get-started-hcp/hcp-gs-deploy]
    - When unauthenticated, the sidebar should show the collection progress element with `x tutorials` and a very light gray background.
- [ ] Confirm that Collection Meta progress display is updated as expected
    - Visit a collection view, such as [/consul/tutorials/get-started-hcp][/consul/tutorials/get-started-hcp]
    - When unauthenticated, the progress section should show the collection progress element with `x tutorials` and a very light gray background.

## 💭 Anything else?

Not at the moment!

[preview]: https://dev-portal-git-zstweak-collection-progress-group-hashicorp.vercel.app
[task]: https://app.asana.com/0/1202097197789424/1203111459158271/f
[figma]: https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=15065%3A191390
[/consul]: https://dev-portal-git-zstweak-collection-progress-group-hashicorp.vercel.app/consul
[/consul/tutorials/get-started-hcp/hcp-gs-deploy]: https://dev-portal-git-zstweak-collection-progress-group-hashicorp.vercel.app/consul/tutorials/get-started-hcp/hcp-gs-deploy
[/consul/tutorials/get-started-hcp]: https://dev-portal-git-zstweak-collection-progress-group-hashicorp.vercel.app/consul/tutorials/get-started-hcp
